go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/edgexfoundry/device-sdk-go v1.1.0
 	github.com/edgexfoundry/go-mod-core-contracts v0.1.31
 	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/golang/snappy v0.0.1 // indirect
```

Comment:
this is strange, any idea why this is pulled in here? I don't see it in the attribution.txt file as well (which probably means we don't use this code in our usage of 3rd party libs, and it's a transitive dependency, but still odd)

Reply:
Looks like it's brought in by the SQL support:
```
$ go mod why github.com/golang/snappy
# github.com/golang/snappy
github.com/edgexfoundry/device-virtual-go/driver
modernc.org/ql/driver
modernc.org/ql
modernc.org/lldb
modernc.org/zappy
modernc.org/zappy.test
github.com/golang/snappy
```

Reply:
Weird, but okay, not an issue just curious

Reply:
thanks for looking into that